msg/esc_report.msg

```diff
@@ -3,7 +3,7 @@ uint32 esc_errorcount					# Number of reported errors by ESC - if supported
 int32 esc_rpm						# Motor RPM, negative for reverse rotation [RPM] - if supported
 float32 esc_voltage					# Voltage measured from current ESC [V] - if supported
 float32 esc_current					# Current measured from current ESC [A] - if supported
-uint8 esc_temperature					# Temperature measured from current ESC [degC] - if supported
+int16 esc_temperature					# Temperature measured from current ESC [centiDegC] - if supported
```

Comment:
Can we keep this SI at the uORB level? Using a float32 is fine unless you think int8 is sufficient (127 C max might not be quite enough for ESCs).